etl/helpers.py

```diff
@@ -1048,3 +1058,51 @@ def create_dag_archive_file(dag_file_archive: Path) -> None:
     dag_file_archive_relative = dag_file_archive.relative_to(Path(paths.DAG_DIR).parent)
     with open(paths.DAG_ARCHIVE_FILE, "a") as file:
         file.write(f"{' ' * n_spaces_include_section}- {dag_file_archive_relative}\n")
+
+
+class DatasetAndExplorer:
+    def __init__(self, dataset, explorer):
+        self.dataset = dataset
+        self.explorer = explorer
+
+    def save(self):
+        # Save ETL dataset to disk.
+        self.dataset.save()
+        if EXPLORER:
```

Comment:
This didn't work for me. You're changing the env dynamically in `command.py`, but this value is from `config.py` when modules were loaded. You should use `os.environ.get("EXPLORER")` instead (that fixed it for me).

Reply:
That's odd, the current implementation works for me.
![Screenshot 2024-07-29 at 10 22 03](https://github.com/user-attachments/assets/01923ef6-c7c1-414a-9d48-968e9f9e053d)
But your suggestion works too, so I'll implement your suggestion (and remove EXPLORER from config). At some point we can refactor to handle this properly.